### PR TITLE
Fixed go lint error using golangci-lint tool.

### LINKED ISSUE #3426: Fix lint errors in go
/kind bug

**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]
There are quite few lint errors in go code that has to be fixed. The error could show up when someone makes an unrelated changes in the file.

**What did you expect to happen:**
All lint errors in go code has to be fixed

### PR DESCRIPTION
**Pull request by andyi2it**
**What this PR does / why we need it**:
Fixes go lint errors

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #3426

**Type of changes**
Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/agent/main.go [hunk 1]
```diff
@@ -36,7 +36,10 @@ import (
 	"github.com/pkg/errors"
 	flag "github.com/spf13/pflag"
 	"go.uber.org/zap"
-	network "knative.dev/networking/pkg"
+
+	// network "knative.dev/networking/pkg"
```
Comment (yuzisun) [inline cmd/agent/main.go]: Can remove this line

File: .golangci.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+linters:
+  enable:
+    - errorlint
+    - gosec
+    - prealloc
+    - unconvert
+    - unparam
+    - gofmt
+  disable:
+    - errcheck
```
Comment (terrytangyuan) [inline .golangci.yml]: Any particular reason to disable this?

Comment (andyi2it) [reply .golangci.yml]: This was done when initially error check were not done part of tests. However checks for tests were disabled globally.
I have removed the errcheck from disable.

**Pull request merged**